content/tutorials/provision-dashboards-and-data-sources.md

```diff
@@ -12,42 +12,45 @@ draft: true
 
 {{% tutorials/step duration="1" title="Introduction" %}}
 
-Learn how you can reuse dashboards and data sources across multiple teams, by provisioning Grafana from version-controlled configuration files.
+Learn how you can reuse dashboards and data sources across multiple teams by provisioning Grafana from version-controlled configuration files.
 
 In this tutorial, you'll:
 
-- Provision dashboards
-- Provision data sources
+- Provision dashboards.
+- Provision data sources.
 
 ### Prerequisites
 
 - Grafana 7.0
+- Administrator privileges on the system you are doing the tutorial on
 
 {{% /tutorials/step %}}
 {{% tutorials/step title="Configuration as code" %}}
 
-As the number of dashboards and data sources grows within your organization, manually managing changes can become tedious and error-prone. Encouraging reuse becomes important to avoid multiple teams redesigning the same dashboards.
-
 Configuration as code is the practice of storing the the configuration of your system as a set of version controlled, human-readable configuration files, rather than in a database. These configuration files can be reused across environments to avoid duplicated resources.
 
+As the number of dashboards and data sources grows within your organization, manually managing changes can become tedious and error-prone. Encouraging reuse becomes important to avoid multiple teams redesigning the same dashboards.
+
 Grafana supports configuration as code through _provisioning_. The resources that currently supports provisioning are:
 
 - [Dashboards](https://grafana.com/docs/grafana/latest/administration/provisioning/#dashboards)
 - [Data sources](https://grafana.com/docs/grafana/latest/administration/provisioning/#datasources)
-- [Notification channels](https://grafana.com/docs/grafana/latest/administration/provisioning/#alert-notification-channels)
+- [Alert notification channels](https://grafana.com/docs/grafana/latest/administration/provisioning/#alert-notification-channels)
 
 {{% /tutorials/step %}}
 {{% tutorials/step title="Set the provisioning directory" %}}
 
-Before you can start provisioning resources, Grafana needs to know where to find these config files.
+Before you can start provisioning resources, Grafana needs to know where to find the _provisioning directory_. A provisioning directory NEED DEFINITION/EXPLANATION.
 
-By default, Grafana looks for a provisioning directory in the configuration directory, but you can configure this yourself by setting the `paths.provisioning` property in the main config file:
+By default, Grafana looks for a provisioning directory in the configuration directory (grafana > conf). You can set a different path by setting the `paths.provisioning` property in the main config file: ((Is this something I might want to do as an admin specifically? Like set this path to a network drive folder?))
```

Comment:
You could set it do a network drive folder. For example, folder can be shared among several Grafana installations. When a dashboard is updated and saved to the folder, all the installations will be updated.

Reply:
Got it, updated.